kuma/search/filters.py

```diff
@@ -140,17 +140,21 @@ def filter_queryset(self, request, queryset, view):
         queries = []
 
         for field in self.fields:
-            search_param = view.query_params.get(field)
+            raw_search_param = view.query_params.get(field, '').lower()
```

Comment:
Why do we lowering the params?

Reply:
Because they are lowercased when added to the index in ``kuma/wiki/search.py``.

Reply:
There are more than one analyzer. There are also case insensitive analyzer
https://github.com/mozilla/kuma/blob/master/kuma/wiki/search.py#L156-L163

Reply:
Only the ``case_insensitive_keyword`` analyzer was used on these fields (``kumascript_macros``, ``css_classnames``, ``html_attributes``), not the case-sensitive analyzer. I think this is a simplification, and the filter name should be renamed.